_docs/style/spacing.md

```diff
@@ -3,59 +3,83 @@ title: Spacing
 layout: docs
 category: Style
 usage: >
-  Helix spacing system provides a logical set of values and limited application rules to better enable design and development teams to use spacing deliberately for improved readability and consistency in product.
+  The Helix spacing system provides a set of values you can use when you determine the amount of space between user interface (UI) components. By applying these spacing values consistently you adhere to accessibility guidelines and improve the readability of control panel pages.
```

Comment:
The Helix spacing system provides a set of values you can use to keep consistent spacing of user interface (UI) components. By applying these spacing values consistently, you adhere to Helix accessibility guidelines and can improve the readability of control panel pages.

Reply:
Done.